reliquery/relic.py

```diff
@@ -295,6 +294,42 @@ def get_pandasdf(self, name: str) -> pd.DataFrame:
         pandas_dataframe = pd.read_json(pandas_json)
         return pandas_dataframe
 
+    def add_file_from_path(self, name: str, path: str) -> None:
+        self.assert_valid_id(name)
+
+        with open(path, "rb") as input_file:
+
+            fileSize = getsizeof(input_file)
```

Comment:
We should probably get the size of the file in from the available bytes in the buffer. We saw that the size of the file could be different then the actual size in bytes of the data on the file. Checkout the add_array or add_image methods on Relic, I believe there are examples there.